changelog

```diff
@@ -1,3 +1,20 @@
+Version 0.1.2
+----------------------------
+1. GDPR compliance to Telemetry and Logger
```

Comment:
Please link to this: https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki/Logging